### ISSUE TITLE
How do I add an answer?

### ISSUE THREAD

**Issue by Graham-72** [serundeputy/qa_forum#13]
I am trying your module. I can add questions, also add comments to questions, but how do I add an answer?

**Comment by serundeputy** [comment]
As it currently stands you can add an `Answer` but adding a comment to an answer functionality is not built yet.

![screen shot 2017-01-18 at 12 18 22 pm](https://cloud.githubusercontent.com/assets/6763671/22075024/6d9e04ca-dd78-11e6-9980-38f1c1a0156b.png)

**Comment by Graham-72** [comment]
I am not able to find this 'Add your answer' form. Is it part of a custom form?

**Comment by serundeputy** [comment]
Yes; we need some documentation (there is not documentation yet except for code comments).

If you make a new question say its `nid` comes out to be 3.  Then the module will respond to the path `{my_website}/question/3` with the relevant question (nid = 3), the related answers, and the `Add your Answer` form.

**Comment by Graham-72** [comment]
Thanks - got it. I have a little test running at http://www.oliverhosting.uk/QAquestions where I am using a view to list the questions and another to display the comments. More work is needed on styles.

I would be inclined to have a question id as well as the nid for each question, so that questions are numbered consecutively. What do you think?

**Comment by serundeputy** [comment]
That can work or we just don't expose the nid in the views.

Ultimately we want a user dashboard on the user profile that is a list (perhaps a view) of the questions that the user is following.
